app/utils/__init__.py

```diff
@@ -21,6 +21,7 @@
 
 """Common functions, variables for all kernelci utils modules."""
 
+import errno
```

Comment:
Maybe keep alphabetical order, but that's a very minor thing.

Reply:
corrected